async-query-core/src/main/java/org/opensearch/sql/spark/config/SparkSubmitParameterModifier.java

```diff
@@ -1,11 +1,12 @@
 package org.opensearch.sql.spark.config;
 
-import org.opensearch.sql.spark.asyncquery.model.SparkSubmitParameters;
+import org.opensearch.sql.spark.parameter.SparkSubmitParametersBuilder;
 
 /**
  * Interface for extension point to allow modification of spark submit parameter. modifyParameter
- * method is called after the default spark submit parameter is build.
+ * method is called after the default spark submit parameter is build. To be deprecated in favor of
+ * {@link org.opensearch.sql.spark.parameter.GeneralSparkParameterComposer}
  */
 public interface SparkSubmitParameterModifier {
```

Comment:
When does modifier come into picture? So we have two concepts composer and modifier? Modifier is used after the completion of building spark submit parameters using composer.

Reply:
I am thinking to remove Modifier once I can verify the Composer can satisfy the use cases.